src/main/java/dev/railroadide/railroad/ui/nodes/ProjectListCell.java

```diff
@@ -114,7 +126,23 @@ protected void updateItem(Project project, boolean empty) {
             nameLabel.setText(project.getAlias());
             pathLabel.setText(project.getPathString());
             lastOpenedLabel.setText(StringUtils.formatElapsed(project.getLastOpened()));
+
+            // Populate Facet Tags
+            for (Facet<?> facet : project.getFacets()) {
+                if (facet != null && facet.getType() != null) {
+                    // Use the FacetType's name for the tag
+                    Label tagLabel = new Label(facet.getType().name()); 
+                    tagLabel.getStyleClass().add("project-list-facet-tag");
+                    tagLabel.getStyleClass().add("facet-" + facet.getType().id());
+                    String description = facet.getType().description();
+                    if (description != null && !description.isBlank()) {
+                        javafx.scene.control.Tooltip.install(tagLabel, new javafx.scene.control.Tooltip(description));
```

Comment:
If you use AI, please at least make sure you are fixing the fully qualified names.

```suggestion
                        Tooltip.install(tagLabel, new Tooltip(description));
```